File: app/src/main/java/com/odoo/base/addons/abirex/dto/Partner.java
package com.odoo.base.addons.abirex.dto;

import com.odoo.core.orm.OValues;

import org.jetbrains.annotations.NotNull;

public class Partner implements DTO {

    int serverId;
    String displayName;
    Company company;

    public Partner(int id, int serverId, String displayName) {
        this.id = id;
        this.serverId = serverId;
        this.displayName = displayName;
        this.company = company;
    }

    int id;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getServerId() {
        return serverId;
    }

    public void setServerId(int serverId) {
        this.serverId = serverId;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    @NotNull
    @Override
    public OValues toOValues() {
        return new OValues();
    }
}

File: app/src/main/java/com/odoo/odoorx/core/data/dto/Partner.kt
package com.odoo.odoorx.core.data.dto

import com.odoo.odoorx.core.base.orm.OValues

open class Partner(var id: Int, var serverId: Int?, var displayName: String?, var email: String, var phone: String, var company: Company?
                   , var address: String, var state: State?, var locality: String) : DTO {

    var fullAddress: String = ""
    var fullContact = ""
    var accountRececivable: Account? = null
    override fun toOValues(): OValues {
        return OValues()
    }

}
